_data/pwnsoni/pwnsoni.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Pawan Soni # No longer than 28 characters
+institution: JSS Academy of Technical Education, Noida 🚩 # no longer than 58 characters
+profile_pic: pic.png # Name and extension of your profile picture(ex. mona.png) The picture must be squared and 544px on width and height.
+quote: "Never say never" # no longer than 100 characters, avoid using quotes(") to guarantee the format remains the same.
```

Comment:
Remove double quotes from begin and end.

Reply:
Updated the required changes.